<ui version="4.0" >
 <class>WidgetTracker</class>
 <widget class="QWidget" name="WidgetTracker" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Trackers</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>4</number>
   </property>
   <property name="leftMargin" >
    <number>14</number>
   </property>
   <property name="topMargin" >
    <number>14</number>
   </property>
   <property name="rightMargin" >
    <number>14</number>
   </property>
   <property name="bottomMargin" >
    <number>14</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="fServers" >
       <property name="font" >
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fBtnRefresh" >
       <property name="text" >
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="font" >
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Filter:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fFilter" >
       <property name="font" >
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="fList" >
     <property name="font" >
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fProgress" >
     <property name="font" >
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>%1 servers registered with tracker</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
